pkg/app/api/grpcapi/piped_api.go

```diff
@@ -25,6 +25,8 @@ import (
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/status"
 
+	"github.com/google/uuid"
```

Comment:
Should be in the same import group.

Reply:
Addressed by https://github.com/pipe-cd/pipe/pull/2815/commits/8790eda637f06196a46a23b46f044bc95d1bb458